docs/tutorials/eda/eda-auto-quick-fit.md

```diff
@@ -0,0 +1,73 @@
+# EDA: Automated Quick Model Fit
+
+The purpose of this feature is to provide a quick and easy way to obtain a preliminary understanding of the
+relationships between the target variable and the independent variables in a dataset.
+
+This functionality automatically splits the training data, performs a simple regression or classification model to the
+data and generating insights: model performance metrics, feature importance and prediction result insights.
+
+To inspect the prediction quality, a confusion matrix displayed for classification problems and scatter plot for
+regression problems. Both representation allow to see the difference between actual and predicted values.
+
+The insights highlight two subsets of the model predictions:
+
+- predictions with the largest classification error. Rows listed in this section are candidates for inspection why the
+  model made the mistakes
+- predictions with the least distance from the other class. Rows in this category are most 'undecided'. They are useful
+  as an examples of data which is close to a decision boundary between the classes. The model would benefit from having
+  more data for a similar cases.
+
+## Classification Example
+
+We will start with getting titanic dataset and performing a quick one-line overview to get the information.
+
+```{.python .input}
+import pandas as pd
+import autogluon.eda.auto as auto
+
+df_train = pd.read_csv('https://autogluon.s3.amazonaws.com/datasets/titanic/train.csv')
+df_test = pd.read_csv('https://autogluon.s3.amazonaws.com/datasets/titanic/test.csv')
+target_col = 'Survived'
+
+auto.quick_fit(df_train, target_col, show_feature_importance_barplots=True)
+```
+
+## Regression Example
+
+In the previous section we tried a classification example. Let's try a regression. It has a few differences.
+We are also going to store the fitted model by specifying `return_state` and `save_model_to_state` parameters.
+This will allow us to use the model to predict test values later.
+
+It is a large dataset, so we'll just keep only few columns for this tutorial.
```

Comment:
"we'll just keep a few columns" or "we'll keep only a few columns"

Reply:
changed to `we'll keep only a few columns`